tests/test_storage.py

```diff
@@ -301,3 +312,78 @@ def test_is_managed_path():
     assert not client.is_managed_path("storage:ubuntu/file.txt")
     assert client.is_managed_path("s3_models:ubuntu/file.txt")
     assert client.is_managed_path("s3_test:")
+
+def test_push_and_delete_cm(request, storages, storage_id):
```

Comment:
You should rename this to something like: `test_cm_storage` to be consistence with existing tests like: `test_local_storage`